packages/core/test/match.test.ts

```diff
@@ -338,6 +338,8 @@ describe('matches() method', () => {
 
       // never called - it's okay if the name is undefined
       expect(name).toBeTruthy();
+    } else if (init.matches('loading')) {
```

Comment:
this test is supposed to test if things compile OK - would be good to add some comment here, inside that if statement to explain what is being tested in here, what specific use case